packages/internals/src/engine-commands/getEnginesMetaInfo.ts

```diff
@@ -145,7 +145,16 @@ export async function resolveEngine(binaryName: BinaryType): Promise<EngineInfo>
   )()
 
   const engineInfo: EngineInfo = {
-    path: enginePathEither,
+    path: pipe(
+      enginePathEither,
+      E.fold(
+        (error) => E.left(error),
+        (path) => {
+          const absolutePath = `${process.cwd()}/${path}`
```

Comment:
you might want to do join here instead of forwardSlash as it'd break on Windows.